server.go

```diff
@@ -252,6 +276,27 @@ func (s *Server) HandleMetricPacket(packet []byte) {
 	}
 }
 
+func (s *Server) HandleTracePacket(packet []byte) {
+	// Unlike metrics, protobuf shouldn't have an issue with 0-length packets
+	if len(packet) == 0 {
+		log.Error("received zero-length trace packet")
+	}
+
+	// Technically this could be anything, but we're only consuming trace spans
+	// for now.
+	newSample := &ssf.SSFSample{}
+	err := proto.Unmarshal(packet, newSample)
+	if err != nil {
+		log.Fatal("Trace unmarshaling error: ", err)
+	}
+
+	s.statsd.Count("veneur.traces.handled", 1, []string{}, 1.0)
```

Comment:
Oh, this is totally superfluous - I only added this for debugging/testing, to ensure that the traces were getting sent. (ie, to determine if a trace isn't showing up because we're not sending it, or because Datadog is not sampling it). I plan to remove this - I'll add a comment to that effect.